src/fde.cc

```diff
@@ -6,10 +6,11 @@
  * Please see the COPYING and CONTRIBUTORS files for details.
  */
 
-/* DEBUG: none          FDE */
+/* DEBUG: section 05    Comm */
```

Comment:
Please leave the "FDE" label, this is not comm.cc or libcomm even though related. We can have multiple labels and it helps document the variety of things the section covers.